Assets/Mirror/Runtime/Transport/MultiplexTransport.cs

```diff
@@ -10,6 +10,8 @@ public class MultiplexTransport : Transport
     {
         public Transport[] transports;
 
+        private Transport available;
```

Comment:
we should do multiplexer '.available' in a separate PR first, that's less confusing

Reply:
Good idea,  see #1213 
Once merged I will fix whatever conflict it causes.